apps/etl/extraction/sources/pdc/extract.py

```diff
@@ -63,234 +76,116 @@ class PdcHazardInputMetadata(pydantic.BaseModel):
-                instance.metadata = params
-                instance.save(update_fields=["resp_code", "metadata"])
-
-                response_data = cls._save_response_data(instance, response)
-
-                if response_data:
-                    cls._update_instance_status(instance, ExtractionData.Status.SUCCESS)
-                    logger.info(
-                        "Data extracted successfully",
-                        extra=log_extra(
-                            {
-                                "extraction_id": instance.id,
-                                "hazard_count": len(response_data),
-                            }
-                        ),
-                    )
-
-                    for hazard in response_data:
-                        cls.process_hazard(instance, hazard)
-                else:
-                    cls._update_instance_status(
-                        instance,
-                        ExtractionData.Status.SUCCESS,
-                        ExtractionData.ValidationStatus.NO_DATA,
-                        update_validation=True,
-                    )
-                    logger.warning(
-                        "No hazard data found in response",
-                        extra=log_extra(
-                            {
-                                "extraction_id": instance.id,
-                            }
-                        ),
-                    )
-                    break  # no data, stop paginating
-
-                # Go to next page
-                page += 1
-                params["pagination"]["page"] = page
-
-            except requests.exceptions.RequestException as e:
-                cls._update_instance_status(instance, ExtractionData.Status.FAILED)
-                logger.error(
-                    "Extraction failed due to request error",
-                    exc_info=True,
-                    extra=log_extra(
-                        {
-                            "extraction_id": instance.id,
-                            "error": str(e),
-                        }
-                    ),
-                )
-                raise
+                ),
+                parent_extraction=self.extraction_object.parent,
+                add_to_queue=False,
+            )
+            chord(
+                [PDCExtractionV2.task.si(exposure_extraction_obj.id)],
+                PDCTransformHandler.task.si(exposure_extraction_obj.id),
+            ).apply_async()
+
+    def handle_polygon(self):
+        self._extraction_fetch_url(
+            self.extraction_metadata.url,
+        )
+
+    def handle_exposure_detail(self):
+        self._extraction_fetch_url(
+            self.extraction_metadata.url,
+            headers={
+                "Authorization": f"Bearer {etl_config.PDC_SENTRY_AUTHORIZATION_KEY}",
+                "Content-Type": "application/json",
+            },
+        )
+
+    def handle_extract(self):
```

Comment:
Let's use @typing.override to flag functions which are override from base

```suggestion
    @typing.override
    def handle_extract(self):
```